config/templates/cluster.yaml

```diff
@@ -469,6 +469,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #     # See https://github.com/coreos/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
+#    # When enabled it will grant sts:assumeRole permission to everyrole in the IAM Worker Role.
+#    # if you specify workerNamedIAMRole the role created will not suffix the random id at the end
+#    # which is useful to update trust policy in target IAM roles that container wants to use through kube2iam.
+#    kube2IamSupport:
+#      enabled: true
+#      workerNamedIAMRole: "yournamedrole"
```

Comment:
It'll work either way so I don't mind, just the same as @pieterlange I'm wondering what the use case is for allowing it to be changed rather than just fixing it against the stack name.